src/configuration/services/solr.md

```diff
@@ -76,4 +76,12 @@ mysearch:
         core_config: !archive "<directory>"
 ```
 
-The `directory` parameter points to a directory in the Git repository, in or below the `.platform/` folder. This directory needs to contain everything that Solr 4.10 needs to start a core. At the minimum, `solrconfig.xml` and `schema.xml`.
+The `directory` parameter points to a directory in the Git repository, in or below the `.platform/` folder. This directory needs to contain everything that Solr 4.10 needs to start a core. At the minimum, `solrconfig.xml` and `schema.xml`.  For example, place them in `.platform/solr/conf/` such that the `schema.xml` file is located at `.platform/solr/conf/schema.xml`.   You can then reference that path like this -
+
+```yaml
+mysearch:
+    type: solr:4.10
+    disk: 1024
+    configuration:
+        core_config: !archive "solr/conf"
```

Comment:
The !archive isn't mentioned above.  What's that mean?

Similarly, the text above says directory, but there's no `directory` parameter here.